compute/instances/troubleshooting/reboot-from-faulty-kernel.mdx

```diff
@@ -0,0 +1,110 @@
+---
+meta:
+  title: Rebooting from a faulty kernel
+  description: This page helps you troubleshoot issues related to the installation of a faulty kernel
+content:
+  h1: Rebooting from a faulty kernel
+  paragraph: This page helps you troubleshoot issues related to the installation of a faulty kernel
+tags: kernel faulty-kernel reboot bugfix
+dates:
+  validation: 2022-11-23
+  posted: 2022-11-23
+categories:
+  - compute
+---
+
+In rare cases your system does fails to boot or boots incorrectly after the installation of a new kernel version, you can use [rescue mode](/compute/instances/concepts/#rescue-mode) to boot into the previous working kernel.
```

Comment:
[Rescue mode](/compute/instances/concepts/#rescue-mode) allows you to boot your Instance into a temporary operating system, often the previous version of the working kernel. Rescue mode is often used to diagnose and resolve issues such as networking issues, firewall configurations...etc.